modules/installation-configuration-parameters.adoc

```diff
@@ -803,6 +806,10 @@ in vSphere.
 |====
 |Parameter|Description|Values
 
+|`platform.vsphere.clusterOSImage`
+|The location from which the installer downloads the {op-system} image. You must set this parameter to perform an installation in a restricted network.
+|An HTTP or HTTPS URL, optionally with a SHA-256 checksum. For example, `\https://mirror.openshift.com/images/rhcos-<version>-vmware.qcow2.gz?sha256=<compressed_sha256>`.
```

Comment:
The rhcos image name for vsphere is like "rhcos-<version>-vmware.<architecture>.ova", for example: rhcos-48.83.202103221318-0-vmware.x86_64.ova